MaterialDesignThemes.Wpf/PackIcon.cs

```diff
@@ -17,9 +17,11 @@ public class PackIcon : PackIconBase<PackIconKind>
         static PackIcon()
         {
             DefaultStyleKeyProperty.OverrideMetadata(typeof(PackIcon), new FrameworkPropertyMetadata(typeof(PackIcon)));
-        }     
+        }
 
         public PackIcon() : base(PackIconDataFactory.Create)
-        { }    
+        {
+            FlowDirection = FlowDirection.LeftToRight;
```

Comment:
But this should be done in the theme style, so it's even possible to change it...